api/dimensions.go

```diff
@@ -32,6 +32,12 @@ func (api *DatasetAPI) getDimensions(w http.ResponseWriter, r *http.Request, lim
 	logData := log.Data{"dataset_id": datasetID, "edition": edition, "version": version, "func": "getDimensions"}
 	var err error
 
+	versionNumber, err := models.ValidateVersionNumber(ctx, version)
+	if err != nil {
+		log.Event(ctx, "invalid version request", log.ERROR, log.Error(err), logData)
+		return nil, 0, err
```

Comment:
We should add the error handling here, so that the response status is `400 Bad Request`:
```
		handleDimensionsErr(ctx, w, "invalid version", err, logData)
```

Reply:
Thank you for spotting this